Initial version of the abstract. (#29)
scionassociation/scion-research_I-D: draft-meynell-panrg-scion-research-questions.md

        org: Armasuisse
--- abstract

This draft intends to summarize all SCION related questions which are deemed important to be answered
for SCION to properly work at Internet scale.
The set of questions is at the moment not comprehensive, but we intend them to initiate a discussion
to determine which questions should remain here, and which ones are missing.

When appropriate, some example solutions can be provided for the community to determine whether said
solutions are adequate or not.

> Whenever a topic spawns a non-trivial discussion, we plan to initiate a new Internet Draft on said topic.

Reviewer: ```suggestion
```

As you see writing a draft is a lot of effort.. I'd not say anything specific about this in the abstract

> This draft intends to summarize all the SCION related questions which are deemed important to answer
> prior for SCION to properly work at Internet scale.
> The set of questions is at the moment not comprehensive, but we intend it to initiate the discussion
> to determine which questions should remain here, and which ones are missing.
> 
> When appropriate, some example solutions are provided, for the community to determine whether said
> solutions are adequate or not.
> 
> Whenever a topic spawns a non-trivial discussion, we plan to initiate a new Internet Draft on said topic.

Reviewer 2: ```suggestion
This draft intends to summarize all SCION related questions which are deemed important to be answered
for SCION to properly work at Internet scale.
The set of questions is at the moment not comprehensive, but we intend them to initiate a discussion
to determine which questions should remain here, and which ones are missing.

When appropriate, some example solutions can be provided for the community to determine whether said
solutions are adequate or not.

Whenever a topic spawns a non-trivial discussion, we plan to initiate a new Internet Draft on said topic.
```
